templates/e-commerce/partials/cart-summary.snip.html

```diff
@@ -0,0 +1,15 @@
+<section class="px2 pt3 pb4 md-px3 md-pt4 col-12 md-col-4 md-commerce-panel-filled commerce-cart-summary">
+  <h2 class="h5 md-mt3">Order Summary</h2>
+  <dl class="flex flex-wrap">
+    <dt class="col-10">Subtotal</dt>
+    <dd class="m0 col-2 pb1 right-align">&#163;1018</dd>
+    <dt class="col-10 pb1">Shipping</dt>
+    <dd class="m0 col-2 pb1 right-align">&#163;25</dd>
+    <dt class="commerce-cart-total col-10 pt2 pb2 bold">Total</dt>
+    <dd class="commerce-cart-total m0 col-2 pt2 pb2 right-align bold">&#163;1043</dd>
+  </dl>
+  <p class="pb3">Donec id elit non mi porta gravida at eget metus. Donec ullamcorper nulla non metus auctor fringilla.</p>
+  <div class="commerce-cart-button center">
```

Comment:
call this commerce-cart-actions 

just in case you want to add more buttons and links i would not call this a button